backend/backend/core/agents.py

```diff
@@ -28,12 +33,13 @@ def __init__(self, model, python_tool):
         super().__init__()
         prompt = ChatPromptTemplate.from_template(
             """
```

Comment:
The prompt template has been modified, but the changes are not reflected in the comment. Please update the comment to match the new template.